docs/proposals/assets/multiple-dns-provider-support/google/google.md

```diff
@@ -0,0 +1,107 @@
+## Google
+
+https://console.cloud.google.com/net-services/dns/zones
+
+Google supports Weighted(Weighted Round Robin) and Geolocation routing policies https://cloud.google.com/dns/docs/zones/manage-routing-policies. Both of these can be configured directly on records in Google Cloud DNS and no secondary Traffic Management resource is required.
+
+Notes:
+
+* Record sets are modelled as a single endpoint with routing policy embedded. This is a different approach to Route53 where each individual A/CNAME would have its own record entry. 
+* Weight must be an integer between 0 - 10000
+* There are no continent options for region, only finer grained regions such as us-east1, europe-west-1 etc...
```

Comment:
This probably presents a problem for supported attributes on clusters? I wonder if we should we come a level and support something like `kuadrant.io/attribute-geo-region=us-east1` ? not part of this proposal but might be a more user friendly approach than continent?

Reply:
Maybe you're saying the same thing here, perhaps this can be pushed back to the user to decide.
For example, they label their ManagedClusters with a geo label.
The value of that label is what get's used for setting up geo with a provider.

Reply:
yeah exactly. It's a bit of a shame they have to know specifics like that but not sure I see any other way

Reply:
I think we should just define our own and map that to a provider specific code. I guess we could optionally allow that to be overridden by additional provider specific attributes if they know what they are doing.